### DISCUSSION TITLE
Using debit card on CardPayment bricks gives an error when try to pay

### DISCUSSION THREAD

**Discussion by joxpulp** [mercadopago/sdk-react#35] [Q&A] [answered]
Im testing a payment with a Visa Debit, Master Debit Card and i get this error on the console
<img width="518" alt="Captura de Pantalla 2023-04-20 a la(s) 05 02 00 p  m" src="https://user-images.githubusercontent.com/73070103/233475321-a558fb64-1f7f-4541-b2a4-bb4281ff22bc.png">

And event if i completed the required information in Bricks and I execute getFormData() I receive this error too:

<img width="517" alt="Captura de Pantalla 2023-04-20 a la(s) 05 05 14 p  m" src="https://user-images.githubusercontent.com/73070103/233475983-ea8ee394-eaab-40e9-9bf9-b823b5fec5d8.png">

The only way to make form work was with a Visa with a bin like this: 40403100 and a payment_method_id: 'credit_card', so i issume this is a problem with debit_cards

**Comment by francineemilia** [comment]
Hi @joxpulp,

Could you send us the bin cards - only the first 6 digits - that  show the console error?

Regards,

**Reply by joxpulp** [reply]
I just found the problem, dont know how to solve it, but it goes like this: if the amount in initialization object is 1 or 2, the first error is shown, if the amount >= 3 it does not happen. 

Bins: 

- 41119720
- 45176601


[I created a sandbox to test it ](https://codesandbox.io/p/github/joxpulp/mercadopago-nextjs/draft/great-frost?workspaceId=d98248d8-98e5-4ed0-bafa-39cf1dd03511&file=%2Fpages%2Fbricks_react.tsx&selection=%5B%7B%22endColumn%22%3A22%2C%22endLineNumber%22%3A26%2C%22startColumn%22%3A22%2C%22startLineNumber%22%3A26%7D%5D)

**Reply by francineemilia** [reply]
@joxpulp I'm trying to simulate the situation you reported, but I'm not having success. I believe that the sandbox you created had an expiration date and I ended up missing it. 

Based on your feedback regarding the transaction amount, I understand that what is happening is that the available payment methods for this public key have a minimum transaction amount greater than 3.

Since this behavior can happen we have developed an improvement regarding this messaging and behavior. If I'm right about this situation, the improvement in question that spells out the situation for you (developer) will be released soon.

Can we confirm this hypothesis once the release happens? The change is expected to be available by 4/26.

Thanks in advance.

**Reply by joxpulp** [reply]
So sorry for that, here is a working link of the [sandbox](https://codesandbox.io/p/github/joxpulp/mercadopago-nextjs/main?layout=%257B%2522activeFilepath%2522%253A%2522%252Fpages%252Fbricks_react.tsx%2522%252C%2522openFiles%2522%253A%255B%2522%252FREADME.md%2522%252C%2522%252Fpages%252Fbricks_react.tsx%2522%255D%252C%2522sidebarPanel%2522%253A%2522EXPLORER%2522%252C%2522gitSidebarPanel%2522%253A%2522COMMIT%2522%252C%2522fullScreenDevtools%2522%253Afalse%252C%2522rootPanelGroup%2522%253A%257B%2522type%2522%253A%2522PANEL_GROUP%2522%252C%2522panels%2522%253A%255B%257B%2522type%2522%253A%2522PANEL%2522%252C%2522panelType%2522%253A%2522TABS%2522%252C%2522id%2522%253A%2522clgvbabz200773b6mdii0ahrx%2522%257D%255D%252C%2522direction%2522%253A%2522vertical%2522%252C%2522id%2522%253A%2522DEVTOOLS_PANELS%2522%252C%2522sizes%2522%253A%255B100%255D%257D%252C%2522tabbedPanels%2522%253A%257B%2522clgvbabz200773b6mdii0ahrx%2522%253A%257B%2522id%2522%253A%2522clgvbabz200773b6mdii0ahrx%2522%252C%2522activeTabId%2522%253A%2522clgvbadbi00bi3b6mxac7e3hq%2522%252C%2522tabs%2522%253A%255B%257B%2522id%2522%253A%2522clgvbabz200763b6md6rbqrbz%2522%252C%2522type%2522%253A%2522TASK_LOG%2522%252C%2522taskId%2522%253A%2522dev%2522%257D%252C%257B%2522type%2522%253A%2522TASK_PORT%2522%252C%2522taskId%2522%253A%2522dev%2522%252C%2522port%2522%253A3000%252C%2522id%2522%253A%2522clgvbadbi00bi3b6mxac7e3hq%2522%252C%2522path%2522%253A%2522%252Fbricks_react%2522%257D%255D%257D%257D%252C%2522showSidebar%2522%253Atrue%252C%2522showDevtools%2522%253Atrue%252C%2522sidebarPanelSize%2522%253A15%252C%2522editorPanelSize%2522%253A47.77777777777777%252C%2522devtoolsPanelSize%2522%253A35%257D), the path of the file is this one: /pages/bricks_react.tsx.

Perfect, waiting on the new release on 4/26 :D thanks

**Reply by francineemilia** [reply, marked as answer]
@joxpulp I have some news about the question you brought up.

The release was brought forward, so part of the problem was mitigated. But there are some situations where the behavior will continue to occur.

Therefore, the recommendation is that you use 3 as a minimum test transaction amount and use our [test cards](https://www.mercadopago.com.ar/developers/es/docs/checkout-bricks/integration-test/test-cards) to test your payment process.

Any questions, we remain at your disposal

**Reply by joxpulp** [reply]
Thanks you for the following on this discussion :D, Oh I can tell that the minimum amount is per country, thanks to clarify this to me
